### PR TITLE
[enterprise-4.4] Update UPI templates to auto-update by including raw files

### PR DESCRIPTION
**Pull request by codyhoag**
FYI @vikram-redhat 

UPI-related previews:

- [AWS](http://file.rdu.redhat.com/~choag/090220/upi-template-includes-4.4/installing/installing_aws/installing-aws-user-infra.html)
- [Azure](http://file.rdu.redhat.com/~choag/090220/upi-template-includes-4.4/installing/installing_azure/installing-azure-user-infra.html)
- [GCP](http://file.rdu.redhat.com/~choag/090220/upi-template-includes-4.4/installing/installing_gcp/installing-gcp-user-infra.html)
- [OSP install](http://file.rdu.redhat.com/~choag/090220/upi-template-includes-4.4/installing/installing_openstack/installing-openstack-user.html)
- [OSP uninstall](http://file.rdu.redhat.com/~choag/090220/upi-template-includes-4.4/installing/installing_openstack/uninstalling-openstack-user.html)

### CODE REVIEW

File: modules/installation-deployment-manager-dns.adoc [hunk 1]
```diff
@@ -12,76 +12,5 @@ objects and load balancers that you need for your {product-title} cluster:
...
-            'privateVisibilityConfig': {
-                'networks': [{
-                    'networkUrl': context.properties['cluster_network']
-                }]
-            }
-        }
-    }]
-
-    return {'resources': resources}
+include::https://raw.githubusercontent.com/openshift/installer/release-4.3/upi/gcp/02_infra.py[]
```
Comment (vikram-redhat) [inline modules/installation-deployment-manager-dns.adoc:15]: 4.4?

**Comment by vikram-redhat** [comment]
Some places it is referencing 4.3, instead of 4.4.

File: modules/installation-deployment-manager-dns.adoc (hunk 1 above)
Comment (codyhoag) [reply modules/installation-deployment-manager-dns.adoc]: Yeah, good eye. An erroneous copy/paste after fixing some issues with those GCP templates. Thanks!

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**